salt/android-sdk/init.sls

```diff
@@ -0,0 +1,21 @@
+{% set version_id = '3859397 sha256=444e22ce8ca0f67353bda4b85175ed3731cae3ffa695ca18119cbacef1c1bea0' %}
+{% set version, source_hash = version_id.split() %}
+
+android-sdk-deps:
+    pkg.installed:
+        - name: unzip
+
+android-sdk:
+    file.managed:
+        - name: /usr/local/src/sdk-tools-linux-{{ version }}.zip
+        - source: https://dl.google.com/android/repository/sdk-tools-linux-{{ version  }}.zip
+        - source_hash: {{ source_hash }}
+
+    cmd.watch:
+        - name: unzip -d /opt/android-sdk /usr/local/src/sdk-tools-linux-{{ version }}.zip &&
+                find /opt/android-sdk -type f -exec test -x {} \; -exec chmod 755 {} \; &&
```

Comment:
`find` takes a `-perm` argument, wouldn't that make more sense than forking out to `test`?